modules/customizing-the-global-header/proc-customizing-rhdh-global-header.adoc

```diff
@@ -0,0 +1,38 @@
+[id="customizing-your-product-global-header_{context}"]
+= Customizing your {product} global header
+
+To replace the {product} global header with a custom header of choice, disable the default header.
+
+.Prerequisites
+* You have configured your {product-custom-resource-type} instance.
+* You have configured the support URL in the `app-config.yaml` file.
+* You have installed the notification plugin.
+
+.Procedure
+
+* Install your own header as a dynamic plugin as shown in the following example:
+
+[source,yaml,subs="+attributes,+quotes"]
+----
+# Custom header implementation
+- package: <npm or oci package-reference>
+  disabled: false
+  pluginConfig:
+    dynamicPlugins:
+      frontend:
+        <package_name>:
+          mountPoints:
+            - mountPoint: application/header
+              importName: <Header component name>
+              config:
+                position: above-main-content
+----
+
+* To disable the global header, disable the right plugin in your `app-config.yaml` configuration file as follows:
```

Comment:
Procedure steps should be a numbered list. See [redhat-documentation.github.io/modular-docs#con-creating-procedure-modules_writing-mod-docs](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs) and [redhat-documentation.github.io/modular-docs#modular-docs-procedure-examples](https://redhat-documentation.github.io/modular-docs/#modular-docs-procedure-examples)

```suggestion
. To disable the global header, disable the right plugin in your `app-config.yaml` configuration file as follows:
```

Reply:
Additionally, use the attribute for `app-config.yaml`